lib/constants.js

```diff
@@ -15,118 +15,503 @@ export const FONTS = [
+      keyword: '#cb4b16',
+      meta: '#859900',
+      number: '#d33682',
+      operator: '#6c71c4',
+      string: '#859900',
+      variable: '#b58900'
+    }
   },
   {
     id: 'solarized light',
     name: 'Solarized (Light)',
     link: 'solarized',
-    light: true
+    light: true,
+    highlights: {
+      background: '#fdf6e3',
+      text: '#657b83',
+      property: '#2aa198',
+      tag: '#93a1a1',
+      attribute: '#2aa198',
+      comment: '#586e75',
+      definition: '#2aa198',
+      keyword: '#cb4b16',
+      meta: '#859900',
+      number: '#d33682',
+      operator: '#6c71c4',
+      string: '#859900',
+      variable: '#839496'
+    }
   },
   {
     id: 'tomorrow-night-bright',
-    name: 'Tomorrow Night'
+    name: 'Tomorrow Night',
+    highlights: {
+      background: '#000000',
+      text: '#eaeaea',
+      property: '#99cc99',
+      tag: '#d54e53',
+      attribute: '#99cc99',
+      comment: '#d27b53',
+      definition: '#e78c45',
+      keyword: '#d54e53',
+      number: '#a16a94',
+      string: '#e7c547',
+      variable: '#7aa6da'
+    }
   },
   {
     id: 'twilight',
-    name: 'Twilight'
+    name: 'Twilight',
+    highlights: {
+      background: '#141414',
+      text: '#f7f7f7',
+      tag: '#997643',
+      attribute: '#d6bb6d',
+      comment: '#777',
+      definition: '#607392',
+      keyword: '#f9ee98',
+      meta: '#f7f7f7',
+      number: '#ca7841',
+      operator: '#cda869',
+      string: '#8f9d6a',
+      variable: '#607392'
+    }
   },
   {
     id: 'verminal',
-    name: 'Verminal'
+    name: 'Verminal',
+    highlights: {
+      background: 'rgba(0, 0, 0, 0.85)',
+      text: '#fff',
+      property: '#0af',
+      tag: '#e06c75',
+      attribute: '#d19a66',
+      comment: '#5c6370',
+      definition: '#34B7FF',
+      keyword: '#9AE1FF',
+      meta: '#abb2bf',
+      number: '#d19a66',
+      operator: '#FA78C3',
+      string: '#98c379',
+      variable: '#ff9ba3'
+    }
   },
   {
     id: 'yeti',
     name: 'Yeti',
-    light: true
+    light: true,
+    highlights: {
+      background: '#ECEAE8',
+      text: '#d1c9c0',
+      property: '#a074c4',
+      tag: '#96c0d8',
+      attribute: '#9fb96e',
+      comment: '#d4c8be',
+      definition: '#55b5db',
+      keyword: '#9fb96e',
+      meta: '#96c0d8',
+      number: '#a074c4',
+      operator: '#9fb96e',
+      string: '#96c0d8',
+      variable: '#a074c4'
+    }
   },
   {
     id: 'zenburn',
-    name: 'Zenburn'
+    name: 'Zenburn',
+    highlights: {
+      background: '#3f3f3f',
+      text: '#dcdccc',
+      property: '#dfaf8f',
+      tag: '#93e0e3',
+      attribute: '#dfaf8f',
+      comment: '#7f9f7f',
+      definition: '#dcdccc',
+      keyword: '#f0dfaf',
+      meta: '#f0dfaf',
+      number: '#dcdccc',
+      operator: '#f0efd0',
+      string: '#cc9393',
+      variable: '#dcdccc'
+    }
   }
-]
+].map(t => ({ ...t, highlights: { ...DEFAULT_HIGHLIGHTS, ...(t.highlights || {}) } }))
```

Comment:
Do you still need this line?

Reply:
No I can just go through and fill in the defaults in the themes that need them.